src/controllers/friend.controller.js

```diff
@@ -63,7 +63,7 @@ const listReceivedRequests = catchAsync(async (req, res, next) => {
   res.json({
     message: i18n.translate('friend.listReceivedRequests'),
     statusCode: https.OK,
-    data: { friendRequests, total: friendRequests.length },
+    data: { friendRequests: friendRequests, total: friendRequests.length },
```

Comment:
key trùng value viết là 1 thôi
```suggestion
    data: { friendRequests, total: friendRequests.length },
```